perfkitbenchmarker/linux_benchmarks/cloudsuite_websearch_benchmark.py

```diff
@@ -12,197 +12,129 @@
-
-FABAN_OUTPUT_DIR = posixpath.join(vm_util.VM_TMP_DIR, 'outputFaban')
-
-PACKAGES_URL = 'http://lsi-www.epfl.ch/parsa'
-INDEX_URL = posixpath.join(PACKAGES_URL, 'index')
-SOLR_CONFIG_URL = posixpath.join(PACKAGES_URL, 'solrconfig.xml')
-SCHEMA_URL = posixpath.join(PACKAGES_URL, 'schema.xml')
-SEARCH_DRIVER_URL = posixpath.join(PACKAGES_URL, 'search.tar.gz')
-
-SOLR_PORT = 8983
-
-CLASSPATH = ('$FABAN_HOME/lib/fabanagents.jar:$FABAN_HOME/lib/fabancommon.jar:'
-             '$FABAN_HOME/lib/fabandriver.jar:$JAVA_HOME/lib/tools.jar:'
-             '$FABAN_HOME/search/build/lib/search.jar')
-
-flags.DEFINE_string('cs_websearch_client_heap_size', '2g',
+flags.DEFINE_string('cs_websearch_client_heap_size',
+                    '2g',
                     'Java heap size for Faban client in the usual java format.'
                     ' Default: 2g.')
-flags.DEFINE_string('cs_websearch_server_heap_size', '3g',
+
+flags.DEFINE_string('cs_websearch_server_heap_size',
+                    '3g',
                     'Java heap size for Solr server in the usual java format.'
                     ' Default: 3g.')
-flags.DEFINE_enum('cs_websearch_query_distr', 'Random', ['Random', 'Ziphian'],
+flags.DEFINE_enum('cs_websearch_query_distr',
+                  'Random',
+                  ['Random', 'Ziphian'],
                   'Distribution of query terms. '
                   'Random and Ziphian distributions are available. '
                   'Default: Random.')
-flags.DEFINE_integer('cs_websearch_num_clients', 1,
-                     'Number of client machines.', lower_bound=1)
-flags.DEFINE_integer('cs_websearch_ramp_up', 90,
-                     'Benchmark ramp up time in seconds.', lower_bound=1)
-flags.DEFINE_integer('cs_websearch_steady_state', 60,
-                     'Benchmark steady state time in seconds.', lower_bound=1)
-flags.DEFINE_integer('cs_websearch_scale', 50,
+flags.DEFINE_integer('cs_websearch_num_clients',
+                     1,
+                     'Number of client machines.',
+                     lower_bound=1)
+flags.DEFINE_integer('cs_websearch_ramp_up',
+                     90,
+                     'Benchmark ramp up time in seconds.',
+                     lower_bound=1)
+flags.DEFINE_integer('cs_websearch_ramp_down',
+                     60,
+                     'Benchmark ramp down time in seconds.',
+                     lower_bound=1)
+flags.DEFINE_integer('cs_websearch_steady_state',
+                     60,
+                     'Benchmark steady state time in seconds.',
+                     lower_bound=1)
+flags.DEFINE_integer('cs_websearch_scale',
+                     50,
                      'Number of simulated web search users.',
                      lower_bound=1)
 
+BENCHMARK_NAME = 'cloudsuite_websearch'
+BENCHMARK_CONFIG = """
+cloudsuite_websearch:
+  description: >
+    Run Cloudsuite Web Search benchmark. Specify the number of worker
+    VMs with --num_vms.
+  vm_groups:
+    servers:
```

Comment:
In the current benchmark implementation yes. However, since we are going to extend the benchmark to support multiple servers in the future, I think we should keep it this way.